backend/labels/models.py

```diff
@@ -33,6 +35,7 @@ class Category(Label):
     objects = CategoryManager()
     example = models.ForeignKey(to=Example, on_delete=models.CASCADE, related_name="categories")
     label = models.ForeignKey(to=CategoryType, on_delete=models.CASCADE)
+    question = models.ForeignKey(to=Question, on_delete=models.CASCADE, null=True, blank=True)
 
```

Comment:
Why does the class `Category` also gets the question field too? Because I think only `TextLabel` and `Scale` need to be associated with questions (as they are used for storing the textfield and scale values). I assume this was just for testing purposes to see if the questions table can be retrieved on the front-end?

Reply:
I think the `Category` needs the question field also because then we'll need to map the category with its scale label, right? If we don't have the question_id in table `labels_category` we will not know which scale is for which category XD

Reply:
Okay, I see 😅